src/openforms/submissions/api/validation.py

```diff
@@ -49,6 +50,14 @@ def validate_submission_allowed(self, submission_allowed):
                 _("Submission of this form is not allowed.")
             )
 
+    def validate_contains_blocked_steps(self, value):
+        if value:
+            raise serializers.ValidationError(
+                _(
+                    "Submission of this form is not allowed due to the answers submitted in a step."
+                )
+            )
+
```

Comment:
This can't be triggered by accident, right? Because of
https://github.com/open-formulieren/open-forms-sdk/pull/370
Should this raise `SuspiciousOperation`

Reply:
It shouldn't be, since if the user navigates to the overview (even by changing the URL) the form wouldn't have persisted the step data that causes the form to prevent continuing to the next step. So the user would have to manually make a PUT to persist the step data and then navigate to the overview to submit the form

Reply:
I interpreted this as a safeguard indeed - an HTTP 400 in this case is better than HTTP 500 because the generic error handling in the SDK *should* display more context so the end-user can contact the municipality with actual information rather than a generic error that's hard to trace.